### PR TITLE
1.2.0-snapshot.20200520.4224.0.2af134ca

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: LATEST [hunk 1]
```diff
@@ -1,4 +1,4 @@
-021f4af394ceb34d9cdfb64e6ac9b908ca2da0f8 1.2.0-snapshot.20200513.4172.0.021f4af3
+021f4af394ceb34d9cdfb64e6ac9b908ca2da0f8 1.2.0-snapshot.20200520.4224.0.2af134ca
```
Comment (cocreature) [inline LATEST:1]: why are you not bumping the commit?

Comment (garyverhaegen-da) [reply LATEST:1]: Haven't had my coffee yet I guess. Also, CI caught it! 🎉

**Comment by garyverhaegen-da** [comment]
☝️ restarting build because macOS failed to download Canton. Given how long a release build is, I'd rather not wait for the other two platforms to finish before rerunning.

**Comment by garyverhaegen-da** [comment]
This is not working. **DO NOT MERGE**. See #6042.

**Comment by garyverhaegen-da** [comment]
Rebased on top of #6042. Let's hope it builds the correct commit now.

**Comment by garyverhaegen-da** [comment]
Release script failed because of a typo (see #6046). The failure occurred in a new, additional step at the end, though, so all of the usual release process, including uploading to external repos (Maven Central, NPM) and GitHub releases, completed successfully.

However, because we do care about that one extra step, we have decided to not test this release and instead move on to creating a new one.

**Pull request merged**